apps/web/app/(app)/environments/[environmentId]/Navigation.tsx

```diff
@@ -185,6 +188,15 @@ export default function Navigation({
           href: `/environments/${environment.id}/settings/setup`,
           hidden: widgetSetupCompleted,
         },
+        {
+          icon: LinkIcon,
+          label: "Link Shortener",
+          href: pathname,
+          hidden: widgetSetupCompleted,
```

Comment:
Please remove `hidden: widgetSetupCompleted` as it will hide "Link Shortener" if a users has completed widget setup 😊